suitcase/utils/tests/tests.py

```diff
@@ -88,5 +88,10 @@ def test_fixture(example_data):
 
 
 def test_fixture_with_ignore(example_data):
-    "Exercise the example_data 'ignore' argument that skip some parameters."
+    "Exercise the example_data 'ignore' argument that skips some parameters."
     assert type(example_data(ignore=['bulk_events'])) is list
+
+
+def test_fixture_with_md(example_data):
+    "Exercise the example_data 'md' argument that is passed to the RunEngine."
+    assert type(example_data(md={'user': 'Dan'})) is list
```

Comment:
If it's easy enough, it would be nice to check that the metadata actually makes it into the start documents, if only to future-proof this feature against accidental breakage by subsequent changes.

Reply:
You are right. I thought about this but took the coward's way out. Here is a proposal for something a little better:def test_fixture_with_md(example_data):
```
def test_fixture_with_md(example_data):
    "Exercise the example_data 'md' argument that is passed to the RunEngine."
    collected_documents = example_data(md={'user': 'Dan'})
    assert type(collected_documents) is list
    assert dict(collected_documents)['start']['md']['user'] == 'Dan'
```